tests/e2e/scale/test_scale_small_file_workload.py

```diff
@@ -12,116 +13,125 @@
-    return ret_value
+    def teardown(self):
+        """
+        Teardown the test environment
 
+        """
+        self.small_files.cleanup()
+        self.es.cleanup()
 
-@pytest.fixture(scope="module")
-def scale_leaks(request):
-    """
-    For scale tests, get bytes used by cephfs at the start,
-    and compare with the cephfs usage at the finish.
-    """
+    def get_cephfs_data(self):
+        """
+        Look through ceph pods and find space usage on all ceph filesystem pods
 
-    def teardown():
+        Returns:
+            Dictionary of byte usage, indexed by pod name.
         """
-        After all the ripsaw creation and deltion runs have finished,
-        test to see if there was a leak in storage.
+        ceph_status = self.ceph.exec_ceph_cmd(ceph_cmd="ceph df")
+        ret_value = {}
+        for pool in ceph_status["pools"]:
+            if "cephfilesystem" in pool["name"]:
+                ret_value[pool["name"]] = pool["stats"]["bytes_used"]
+        return ret_value
+
+    def display_ceph_usage(self, msg, data):
         """
-        log.info("In scale_leaks teardown")
-        orig_data = cephfs_data.ceph_fs_use
-        now_data = get_cephfs_data()
-        still_going_down = True
-        while still_going_down:
-            log.info("Waiting for Ceph to finish cleaning up")
-            time.sleep(120)
-            new_data = get_cephfs_data()
-            still_going_down = False
-            for entry in new_data:
-                if new_data[entry] < now_data[entry]:
-                    still_going_down = True
-                    now_data[entry] = new_data[entry]
-        for entry in now_data:
-            # Leak indicated if over %50 more storage is used.
-            log.info(f"{entry} now uses {now_data[entry]} bytes")
-            log.info(f"{entry} originally used {orig_data[entry]} bytes")
-            ratio = now_data[entry] / orig_data[entry]
-            check = ratio < 1.50
-            errmsg = f"{entry} over 50% larger -- possible leak"
-            assert check, errmsg
+        Display the pool usage in a pretty way
 
-    request.addfinalizer(teardown)
+        Args:
+            msg (str): the message string to display with the values
+            data (dict): dictionary of pools -> capacity (in bytes)
 
-    return scale_leaks
+        """
+        log.info(f"The pools usage {msg} is :")
+        for entry in data:
+            log.info(f"{entry} now uses {data[entry]:,} bytes")
 
+    @pytest.mark.parametrize(
+        argnames=["file_size", "files", "threads", "interface"],
+        argvalues=[
+            pytest.param(*[16, 125000, 4, constants.CEPHFILESYSTEM]),
+            pytest.param(*[16, 375000, 4, constants.CEPHFILESYSTEM]),
+            pytest.param(*[16, 625000, 4, constants.CEPHFILESYSTEM]),
+            pytest.param(*[16, 875000, 4, constants.CEPHFILESYSTEM]),
+            pytest.param(*[16, 1250000, 4, constants.CEPHFILESYSTEM]),
```

Comment:
Can we also different file_size? like 8 and 32 ?
pytest.param(*[8, 125000, 4, constants.CEPHFILESYSTEM]),
pytest.param(*[32, 875000, 4, constants.CEPHFILESYSTEM]),
pytest.param(*[32, 1250000, 4, constants.CEPHFILESYSTEM]),

Reply:
sure, i will add more tests.